prefect_aws/ecs.py

```diff
@@ -212,6 +212,15 @@ class ECSTask(Infrastructure):
             "unless `stream_output` is set. "
         ),
     )
+    cloudwatch_logs_options: Optional[Dict[str, str]] = Field(
+        default=None,
+        description=(
+            "When `configure_cloudwatch_logs`, this setting may be used to pass "
+            "additional options to the CloudWatch logs configuration or override "
+            "the default options. See [the AWS documentation](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/using_awslogs.html#create_awslogs_logdriver_options) "  # noqa
```

Comment:
The UI doesn't handle markdown, so we can include the link and remove the markdown formatting , or find a different way to communicate this info.

Reply:
Ah gee. Is it worth just leaving as is since UI markdown support will happen eventually? Markdown _is_ intended to be readable unrendered. I'm down to move the link to the bottom of the doc instead.

Reply:
It's not as pretty, but it does provide more useful info to the user, so I'm good with leaving it as is and letting a future UI update make it look more pretty.